composer.json

```
@@ -9,7 +9,7 @@
     "homepage": "https://laminas.dev",
     "license": "BSD-3-Clause",
     "require": {
-        "php": "^7.3 || ^8.0",
+        "php": "^7.3 || ^8.0 || ^8.1",
```

`^8.0` actually include 8.1, the requirement should be:

```suggestion
        "php": "^7.3 || ~8.0.0 || ~8.1.0",
```

for consistency with other components.

---

You are right.